joplin/js/editor.js

```diff
@@ -94,32 +63,6 @@ $(function() {
       $(label).append(link);
     }
   }
-  // TODO: what is this for??
-  $('.js-proxy-click').click(function() {
```

Comment:
I found that this was for an older implementation of save and preview buttons. they buttons weren't working out of the box so they needed to be tweaked.